test/functional/terminal/tui_spec.lua

```diff
@@ -2736,13 +2737,61 @@ describe('TUI', function()
       unchanged = true,
     }
   end)
+
```

Comment:
Not sure where would be the best place, but can we also add a test case for `v:termrequest` (which is currently missing apparently) --- as well as `args.data`?

Reply:
The new test in `terminal/buffer_spec.lua` tests `v:termrequest`. The new tests in `tui_spec.lua` test `args.data`.